CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Using the Svelte framework
+
+A vast majority of project work relies on HTML and CSS, rather than super-fancy JavaScript. So, you shouldn't need much Svelte knowledge to get up and running! Here are the resources we recommend reading before getting started.
+
+**Basics on using CSS in Svelte:** [Read this CSS tricks article](https://css-tricks.com/what-i-like-about-writing-styles-with-svelte/)
+
+**Fundamentals tutorial:** [Just click through the "Basics" section](https://svelte.dev/tutorial/basics)
+
+**Understanding Sapper**: [Read "Introduction" and "Sapper app structure"](https://sapper.svelte.dev/docs)
+
+## CSS standards
+
+This project is CSS intensive. For this reason, we've worked hard to ensure that all colors, fonts, and sizing values are documented using **CSS variables.**
+
+So, when creating new sections on the site, **check the `:root` ruleset in `global.css` for a list of all variables available.** To get a better understand of what CSS variables are and how they work, [read a getting started guide here](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) or crawl through the existing codebase to see them in action.
+
+### Mobile considerations
+
+Most of our website needs to adjust for mobile layouts. Some recommendations:
+1. Use the breakpoint `max-width: 600px` when writing media queries that target mobile devices. Use your best judgment on whether to use a different breakpoint or multiple breakpoints.
+2. Any content on mobile **should have a gutter / margin of 20px** on either side. In other words, text blocks and headers should have `20px` of breathing room. On desktop, this value is either `30px` or a custom value depending on the context.
+2. No need to handle font-resizing if you're using default values. Check for any `@media` queries already specified in `global.css`.
+
+### When variables don't quite fit
+If you're referencing design guidelines where sizing or margins don't quite match up to our defaults, choose the closest match possible (for example, using a font size of `24px` instead of `26px`). If the closest match still doesn't feel right, do one of the following:
+1. If you're a Bits of Good member, consult a designer through our slack channel **#gt-org-website**
+2. If you're not a member, add a note on the issue for us to review
+3. If you don't find consultation necessary, use a custom value that does not already exist as a CSS variable
+
+## Pull Request Process
+
+1. Make sure you note the issue number the PR resolves using the syntax **Resolves #[issue-number]**
```

Comment:
Note branch and git semantics we want to put into place 
branch naming:  [name]/[issue#]/[branch-purpose] ex. james/63/project-header-revamp
commit: [type]: [subject] ex.  feat: add mobile responsiveness

Like this semantic style: https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

Reply:
Good point! I'll add that